src/_components/card_talk.liquid

```diff
@@ -0,0 +1,20 @@
+<div class="card-talk">
+  {% if show_title %}
+  <div class="card-talk__title">{{ event.title }}</div>
+  {% endif %}
+  <div class="card-talk__speaker">
+    {% if speaker.photo_url %}
+    <figure class="card-talk__profile">
+      <img src="{{ speaker.photo_url }}" class="card-talk__photo" />
+    </figure>
```

Comment:
i think it would look better if people without photos yet had a silhouette placeholder image

Reply:
n/a as everyone now has a photo